install

```diff
@@ -218,10 +211,6 @@ def shell_profile
   end
 end
 
-# Invalidate sudo timestamp before exiting (if it wasn't active before).
-Kernel.system "/usr/bin/sudo -n -v 2>/dev/null"
```

Comment:
```ruby
Kernel.system "/usr/bin/sudo -n -v 2>/dev/null" if mac?
```

Reply:
It is unnecessary because `sudo` will be checked only when it is needed in below.

Reply:
Yep, so disable this line on Linux as above.

Reply:
On Mac, in where `sudo` requires a password,
`sudo -n -v` fails.
Therefore these lines are meaningless `if mac` is added.

Reply:
These lines are from Homebrew/install on macOS.
See https://github.com/Homebrew/install/blob/8e6decdff581a4a463d66c363831f8da6b865ff8/install#L148-L150

Reply:
Sorry, the previous comment is wrong and I had misunderstandings.
This `at_exit` code works as expected and do not needed to be moved,
as `sudo -k` do nothing as you mentioned.

I will change them back as before about `at_exit`.